src/micro/extends.html

```diff
@@ -87,6 +91,10 @@
       if (!p) {
         var np = this.getNativePrototype(tag);
         p = this.extend(Object.create(np), Polymer.Base);
+        var constructor = function(){};
```

Comment:
The function's not returning a prototype anymore. Let's change the name or add a new function?